icons/flame-kindling.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M15.5 7c-1-1-2.5-2-3.5-5-1 0-4 3-1.6 5.6a2 2 0 1 1-3.3 2c-.1.3-.1.6-.1.9A5 5 0 1 0 15.5 7" />
+  <path d="m5 22 14-4" />
+  <path d="m5 18 14 4" />
```

Comment:
I'm still not happy with how blurry the bottom of the flame is, but nonetheless, this one's still over-optimised:

```suggestion
  <path d="M12 2c1 2.98 2.53 3.97 3.5 5a5 5 0 1 1-8.42 2.58 2 2 0 1 0 3.3-2.02A3.55 3.55 0 0 1 12 2" />
  <path d="m5 22 14-4" />
  <path d="m5 18 14 4" />
```

Reply:
This one has a blurry line as well. I've made an optimized one